packages/autocomplete-css/update.js

```diff
@@ -0,0 +1,366 @@
+      we have `manual-property-desc.json` which is a list of manually updated
+      descriptions for properties where there are none. This was a last resort
+      intended to provide the highest quality of completions possible.
+      Overtime many items on this list will likely be able to be removed just as
+      new ones are added. After running the update script you'll see a warning
+      saying how many properties are without completions that would then need to
+      be added to the JSON file.
+
+  "spec-shortname": {
+    "spec": {
+      "title": "",
+      "url": ""
+    },
+    "properties": [
+      {
+        "name": "",
+        "value": "",
+        "initial": "",
+        "appliesTo": "",
+        "percentages": "",
+        "computedValue": "",
+        "canonicalOrder": "",
+        "animationType": "",
+        "media": "",
+        "styleDeclaration": [ "", "", "" ]
+      }
+    ],
+    "atrules": [
+      {
+        "name": "",
+        "descriptors": [
+          {
+            "name": "",
+            "for": "",
+            "value": "",
+            "type": ""
+          }
+        ]
+      }
+    ],
+    "selectors": [],
+    "values": [
+      {
+        "name": "",
+        "type": "",
+        "prose": "Optional description",
+        "value": "",
+        "values": [
+          {
+            "name": "",
+            "prose": "Optional Description",
+            "type": "",
+            "value": ""
+          }
+        ]
+      }
+    ],
+    "warnings": []
+  }
+*/
+
+const css = require("@webref/css");
+const fs = require("fs");
+const CSSParser = require("./cssValueDefinitionSyntaxExtractor.js");
+const manualPropertyDesc = require("./manual-property-desc.json");
+
+async function update(params) {
+  const parsedFiles = await css.listAll();
+
+  const properties = await buildProperties(parsedFiles);
+  const tags = await getTagsHTML();
+  const pseudoSelectors = await getPseudoSelectors();
+
+  const completions = {
+    tags: tags,
+    properties: properties,
+    pseudoSelectors: pseudoSelectors
+  };
+
+  // Now to write out our updated file
+  fs.writeFileSync("completions.json", JSON.stringify(completions, null, 2));
+
+  // Now to determine how many properties have empty descriptions.
+
+  let count = 0;
+  let showEmpty = false;
+
+  for (const param of params) {
+    if (param === "--show-empty") {
+      showEmpty = true;
+    }
+  }
+
+  for (const prop in completions.properties) {
+    if (completions.properties[prop].description === "") {
+      if (showEmpty) {
+        console.log(prop);
+      }
+      count ++;
+    }
+  }
+
+  console.log("Updated all `autocomplete-css` completions.");
+  console.log(`Total Completion Properties without a description: ${count}!`);
+  console.log("It is not required to fix the above empty completions issue.");
+  console.log("Use `node update.js --show-empty` to show the empty property names.");
```

Comment:
Some of these console.log strings are not super relevant, now that there are zero properties with empty descriptions.

Especially the line "It is not required to fix the above empty completions issue." -- now there is no such issue!

This suggestion is to conditionally not print some of the info if the missing descriptions count is zero.

```suggestion
  console.log("Updated all `autocomplete-css` completions.");
  console.log(`Total Completion Properties without a description: ${count}!`);
  if (count !== 0) {
    console.log("It is not required to fix the above empty completions issue.");
    console.log("Use `node update.js --show-empty` to show the empty property names.");
  };
```


Before:

```
$ node update.js
Updated all `autocomplete-css` completions.
Total Completion Properties without a description: 0!
It is not required to fix the above empty completions issue.
Use `node update.js --show-empty` to show the empty property names.
```

After:

```
$ node update.js
Updated all `autocomplete-css` completions.
Total Completion Properties without a description: 0!
```

Reply:
Good call on making this last bit optional, way less verbose when it's not needed